recipes/python-pgplot/conda_build_config.yaml

```diff
@@ -0,0 +1,9 @@
+numpy:
+  - 1.26
+  - 2.0
+c_stdlib:
+  - sysroot                    # [linux]
+  - macosx_deployment_target   # [osx]
+c_stdlib_version:              # [unix]
+  - 2.17                       # [linux and x86_64]
+  - 11.0                       # [osx and arm64]
```

Comment:
Why you need a custom one, can't you just use the official conda-forge-pinning ?